src/mudlet.cpp

```diff
@@ -368,8 +369,14 @@ mudlet::mudlet()
     mpActionDisconnect = new QAction(tr("Disconnect"), this);
     mpActionDisconnect->setObjectName(qsl("disconnect"));
 
+    mpActionCloseProfile = new QAction(tr("Close profile"), this);
+    mpActionCloseProfile->setIcon(QIcon(qsl(":/icons/system-close.png")));
```

Comment:
please rename file to close-profile
```suggestion
    mpActionCloseProfile->setIcon(QIcon(qsl(":/icons/close-profile.png")));
```